docs/guides/core-concepts/retry-ability.mdx

```diff
@@ -146,22 +146,21 @@ it('creates two items', () => {
   cy.get('.new-todo').type('todo A{enter}')
   cy.get('.new-todo').type('todo B{enter}')
 
-  cy.get('.todo-list li') // command
-    .should('have.length', 2) // assertion
+  cy.get('.todo-list li') // query
+    .should('have.length', 2) // query
     .and(($li) => {
-      // 2 more assertions
+      // 2 expect() statements inside of the .and() query
       expect($li.get(0).textContent, 'first item').to.equal('todo a')
       expect($li.get(1).textContent, 'second item').to.equal('todo B')
     })
 })
 ```
 
-Because the second assertion
-`expect($li.get(0).textContent, 'first item').to.equal('todo a')` fails, the
-third assertion is never reached. The command fails after timing out, and the
-Command Log correctly shows that the first encountered assertion
-`should('have.length', 2)` passed, but the second assertion and the command
-itself failed.
+Because the first expect statement
+(`expect($li.get(0).textContent, 'first item').to.equal('todo a')`) fails, the
+second statement is never reached. The command fails after timing out, and the
+Command Log correctly shows that `.get()` and `.should()` passed, but the "first
+item" assertion failed.
```

Comment:
I expected this to say something like the explicit assertion will not cause the .and command to retry"...going into the explanation of the command seems like expected behavior (even though I think we just fixed this).

Reply:
I'm not sure I'm following you here. The `expect()` statement is inside the callback function for `.and()` - it 100% does cause it to retry, and keep retrying until it times out.